datafusion/sql/src/unparser/ast.rs

```diff
@@ -367,6 +367,11 @@ impl RelationBuilder {
         new.relation = Option::Some(TableFactorBuilder::Derived(value));
         new
     }
+    pub fn empty(&mut self) -> &mut Self {
+        let new = self;
+        new.relation = Some(TableFactorBuilder::Empty);
+        new
```

Comment:
I think you could also do this (which is more concise)

```suggestion
        self.relation = Some(TableFactorBuilder::Empty);
        new
```

Though I see it follows the same pattern as the code above so if we are going to change any of the we should change them all